android-wsbridge/app/src/main/java/com/dji/wsbridge/lib/BridgeApplication.java

```diff
@@ -21,10 +23,22 @@ public static BridgeApplication getInstance() {
         return instance;
     }
 
+    public static void logToFirebase(String message) {
```

Comment:
I think these two should be in the Util class as well.

Reply:
Done